packages/api-server/src/base/env-config.ts

```diff
@@ -20,6 +21,11 @@ export const envConfig = {
   logFormat: getOptional("LOG_FORMAT"),
   logRequestBody: getOptional("WEB3_LOG_REQUEST_BODY"),
   port: getOptional("PORT"),
+
+  // default is 128kb, see also https://github.com/ethereum/go-ethereum/blob/b3b8b268eb585dfd3c1c9e9bbebc55968f3bec4b/core/tx_pool.go#L43-L53
+  maxTransactionSize: +(getOptional("MAX_TRANSACTION_SIZE") || "131072"),
```

Comment:
extract `131072` to `/packages/api-server/src/methods/constant.ts`file. and don't overwrite with default value here, `envConfig.maxTransactionSize` should be a optional value. use default value if needed in outer place.